src/pages/App.tsx

```diff
@@ -12,8 +11,9 @@ import React, {
   useMemo,
 } from "react"
 import { Route, Routes } from "react-router-dom"
-import { useDispatch, useSelector } from "react-redux"
+import { styled, useTheme } from "@mui/material"
```

Comment:
do you need styled?

Reply:
Yeah, that was used for the background of the app.